backend/private-graph/graph/auth.go

```diff
@@ -20,15 +20,15 @@ type LoginCredentials struct {
 	Password string `json:"password" binding:"required"`
 }
 
-var ADMIN_PASSWORD = os.Getenv("ADMIN_PASSWORD")
-var ADMIN_PASSWORD_TOKEN_DURATION = time.Now().Add(time.Hour * 24).Unix()
+var AdminPassword = os.Getenv("ADMIN_PASSWORD")
+var ADMIN_PASSWORD_TOKEN_DURATION = time.Hour * 24
```

Comment:
nit: rename to `AdminPasswordTokenDuration` to maintain same style for package variables as the rest of the go codebase.

Reply:
Gotcha. Will take note going forward.